boards/arm/sparkfun_pro_mirco_rp2040/sparkfun_pro_micro.dtsi

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
This file's name should probably have a `_connector` suffix.

Reply:
Fixed.